cmd/selfupdate/help.go

```diff
@@ -0,0 +1,18 @@
+package selfupdate
+
+// Note: "|" will be replaced by backticks in the help string below
+var selfUpdateHelp string = `
+This command downloads the latest release of rclone and replaces
+the currently running binary. The download is verified with a hashsum.
+
+The |--version VER| flag, if given, will update to a concrete version
+instead of the latest one. If you omit micro version from |VER| (for
+example |1.53|), the latest matching micro version will be used.
+
+If you previously installed rclone via a package manager, the package may
+include local documentation or configure services. This command will update
+only the rclone executable so the local manual may become inaccurate after it.
```

Comment:
Note here `--format deb`?

Reply:
Thinking about name for this option I considered `--format` and selected `--package`
What do you think?
@ncw @albertony

Reply:
No strong opinion here. As zip is the default, I'm fine with `--package`.
In general I find that "package" sort of implies something handled by a package manager, but then "zip package" is also sometimes used as a synonym for "zip archive", so `--package zip` is not that strange, perhaps.. Maybe "distribution" is a generic term for different variants of the same release, but it is also a word with strong associations to linux, so..

Reply:
I think I mildly prefer `zip|deb|rpm package` over `zip|deb|rpm format`. I think `package` reads slightly better.

Not sure why I said `--format deb` above - I think the option was called `format` then, or maybe I just got it wrong :-)

I'm happy with `--package`.

My note was do indicate that this para should probably talk about `--package`

> If you previously installed rclone via a package manager, the package may
> include local documentation or configure services. This command will update
> only the rclone executable so the local manual may become inaccurate after it.
> 

Should maybe be

> If you previously installed rclone via a package manager, the package may
> include local documentation or configuration services. You may wish to update with the flag `--package deb` or `--package rpm` (whichever is correct for your OS) to update these too. This command with the default `--package zip` will update
> only the rclone executable so the local manual may become inaccurate after it.
>

Reply:
Done